src/utils/user.utils.ts

```diff
@@ -0,0 +1,28 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import 'dotenv/config';
+import { Iuser } from '../interfaces/user.iterfaces';
+
+const { JWT_SECRET_KEY } = process.env;
+
+export const generateToken = (user: Iuser) => jwt.sign(
+  {
+    id: user.id,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    userName: user.userName,
+    email: user.email,
+    gender: user.gender,
+    roleId: user.roleId,
+    institutionId: user.institutionId,
+  },
+  JWT_SECRET_KEY!,
+  { expiresIn: '1d' },
+);
+
+// export const hashPassword = (password: string) => bcrypt.hashSync(password, 10);
```

Comment:
Could you have forgotten to uncomment this?😆

Reply:
No, I forgot to remove that statement cuz I am not using it.